src/main/java/com/github/technus/tectech/recipe/EyeOfHarmonyRecipe.java

```diff
@@ -8,11 +8,7 @@
 import static gregtech.api.util.GT_Utility.getPlasmaFuelValueInEUPerLiterFromMaterial;
 import static java.lang.Math.min;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.HashMap;
-import java.util.List;
+import java.util.*;
```

Comment:
Spotless is going to be sad on this. It is possible to change at least IntelliJ to not do it in editor cinfig